build.sh

```diff
@@ -55,30 +55,6 @@ check_native_prereqs()
 
 prepare_managed_build()
 {
-    # Pull NuGet.exe down if we don't have it already
-    if [ ! -e "$__nugetpath" ]; then
-        which curl wget > /dev/null 2> /dev/null
-        if [ $? -ne 0 -a $? -ne 1 ]; then
-            echo "cURL or wget is required to build corefx. Please see https://github.com/dotnet/corefx/blob/master/Documentation/building/unix-instructions.md for more details."
```

Comment:
what's the build experience if you don't run init-tools first?  Should we have an echo for this case and a pointer to (updated) documentation still?